libminifi/include/core/ClassLoader.h

```diff
@@ -636,9 +612,9 @@ T *ClassLoader::instantiateRaw(const std::string &class_name, utils::Identifier
 }
 
 }/* namespace core */
-} /* namespace minifi */
-} /* namespace nifi */
-} /* namespace apache */
-} /* namespace org */
+}  // namespace minifi
+}  // namespace nifi
```

Comment:
The `core` namespace comment is still inconsistent.

Reply:
Corrected.